tests/suspend_nodes.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import infra.ccf
+import infra.proc
+import infra.notification
+import infra.net
+import infra.e2e_args
+from threading import Timer
+import time
+import random
+import suite.test_requirements as reqs
+import infra.logging_app as app
+
+from loguru import logger as LOG
+
+# pbft will store up to 34 of each message type (pre-prepare/prepare/commit) and retransmit these messages to replicas that are behind, enabling catch up.
+# If a replica is too far behind then we need to send entries from the ledger, which is one of the things we want to test here.
+# By sending 18 RPC requests and a getCommit for each of them (what raft consideres as a read pbft will process as a write),
+# we are sure that we will have to go via the ledger to help late joiners catch up (total 36 reqs > 34)
+TOTAL_REQUESTS = 9  # x2 is 18 since LoggingTxs app sends a private and a public request for each tx index
+
+s = random.randint(1, 10)
```

Comment:
I would be a tiny bit nicer to make the seed an extra CLI arg, and set the default to the random seed.

Reply:
done